CHANGELOG.md

```diff
@@ -10,48 +11,71 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- Comment editor will be shown properly, when comment had been added on fixed column and some scrolling was performed [#9645](https://github.com/handsontable/handsontable/issues/9645)
-- Fixed `dropdown`-type editor that prevents typing values using IME. [#9672](https://github.com/handsontable/handsontable/issues/9672)
-- Internal: Fixed the broken "verify-bundles" script, used in the version freezing process. [#9891](https://github.com/handsontable/handsontable/issues/9891)
-- *React:* Fixed a problem with the dropdown-typed cells validation when using `HotColumn`. [#10065](https://github.com/handsontable/handsontable/issues/10065)
+For more information on this release, see:
+- [Blog post](https://handsontable.com/blog/articles/handsontable-12.3.0)
+- [Documentation (12.3)](https://handsontable.com/docs/12.3/)
+- [Release notes](https://handsontable.com/docs/release-notes/#_12-3-0)
 
 ### Added
-- Added a new feature that enables the user to copy the contents of column headers by using 3 new context menu options ("Copy with headers", "Copy with group headers", and "Copy headers only"), along with new API methods for using the feature programmatically. [#2288](https://github.com/handsontable/handsontable/issues/2288)
-- Exported the `CellCoords` and `CellRange` classes, to make them part of the public API. [#9755](https://github.com/handsontable/handsontable/issues/9755)
-- Add missing TypeScript definitions for the manual column freeze hooks. [#9859](https://github.com/handsontable/handsontable/issues/9859)
+
+- Added a new feature that lets you copy the contents of column headers by using 3 new context menu options: "Copy with headers", "Copy with group headers", and "Copy headers only". [#2288](https://github.com/handsontable/handsontable/issues/2288)
+- Added 4 new API methods for copying column headers programmatically: `copyCellsOnly()`, `copyWithColumnHeaders()`, `copyWithAllColumnHeaders()`, and `copyColumnHeadersOnly()`. [#2288](https://github.com/handsontable/handsontable/issues/2288)
+- Added missing TypeScript definitions of the `CellCoords` and `CellRange` classes, which are used in the arguments of some of the APIs. [#9755](https://github.com/handsontable/handsontable/issues/9755)
+- Added missing TypeScript definitions for the following Handsontable hooks: `beforeColumnFreeze`, `afterColumnFreeze`, `beforeColumnUnfreeze`, and `afterColumnUnfreeze`. [#9859](https://github.com/handsontable/handsontable/issues/9859)
+
+### Fixed 
+
+- Fixed an issue where the `UndoRedo` plugin didn't work properly with preconfigured columns. [#10108](https://github.com/handsontable/handsontable/pull/10108)
+- Fixed an issue where formulas were not updated after `updateData()` calls. [#10011](https://github.com/handsontable/handsontable/issues/10011)
```

Comment:
I think that would be clearer:

```suggestion
- Fixed an issue where formulas programmatically inserted using `updateData()` where not evaluated. [#10011](https://github.com/handsontable/handsontable/issues/10011)
```

Reply:
Thanks, updated: [8e21d43](https://github.com/handsontable/handsontable/pull/10153/commits/8e21d43c7f33b8b954d0f96018aec53bc8288303)